build.sh

```diff
@@ -2,6 +2,12 @@
 
 set -e
 
+if [ "$(uname)" == "Darwin" ]; then
+    #Mac OS X platform
+	yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
>	 [](start = 0, length = 1)

Consistency with tabs spaces, and general indentation. Note that below you have *two* spaces for indentation. Here you have either *four* spaces (on one of the new lines) or *one* tab (on two of the new lines).